app/scripts/modules/core/pipeline/config/triggers/travis/selectorTemplate.html

```diff
@@ -0,0 +1 @@
+<travis-trigger-options command="vm.command"></travis-trigger-options>
```

Comment:
This should be in-lined as it's one line of HTML rather than creating a separate file.